CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [v0.27.1](https://github.com/nervosnetwork/ckb-sdk-java/compare/v0.27.0...v0.27.1) (2020-2-3)
+
+### Feature
+
+* Add outputsValidator as parameter to sendCapacity rpc ([d6d715e](https://github.com/nervosnetwork/ckb-sdk-java/commit/d6d715e05d0877697f4f78f4f148b74df8d26487))
```

Comment:
It seems to be added in the `sendTransaction` method.

Reply:
Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/280/commits/654820f8f0ff17fd87dd449a3d7d0709a7f2e5fa